src/controller/CHIPDeviceController.h

```diff
@@ -278,19 +265,22 @@ class DLL_EXPORT ChipDeviceController : public SecureSessionMgrCallback, public
     NodeId mLocalDeviceId;
     Inet::IPAddress mDeviceAddr;
     uint16_t mDevicePort;
+    Inet::InterfaceId mInterface;
     Optional<NodeId> mRemoteDeviceId;
     uint32_t mMessageNumber = 0;
 
     SecurePairingSession mPairingSession;
+    SecurePairingUsingTestSecret * mTestPairingSession = nullptr;
```

Comment:
nit: could we update the names a bit? `mTestPairingSession` vs `mCurrentPairingSession` make it sound as they are both of the same type (a pairing sesion) when in fact this is not the case.

How about: `mSecurePairingSecret` and `mSecurePairingSession` ?